app/models/option_set_import.rb

```diff
@@ -199,4 +202,9 @@ def blank_interior_cell?(row)
       !row[row.index(nil)..-1].all?(&:nil?)
     end
 
+    def name_locale_key
+      preferred_locale = configatron.preferred_locales.first
+      name_locale_key = "name_#{preferred_locale}".to_sym
```

Comment:
If we use the new configatron key you can combine these two lines and remove the assignment.